build_projects/dotnet-host-build/MsiTargets.cs

```diff
@@ -162,6 +162,7 @@ public static BuildTargetResult GenerateDotnetHostFxrMsi(BuildTargetContext c)
             var inputDir = c.BuildContext.Get<string>("HostFxrPublishRoot");
             var wixObjRoot = Path.Combine(Dirs.Output, "obj", "wix", "hostfxr");
             var hostFxrBrandName = $"'{Monikers.GetHostFxrBrandName(c)}'";
+            var upgradeCode = Utils.GenerateGuidFromName(HostFxrMsi).ToString().ToUpper();
```

Comment:
What is the name we're using here? Does it contain branding or other things that would trigger an upgrade code change when we don't want one?